clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/DefaultOrchestrationProcessor.groovy

```diff
@@ -118,6 +118,7 @@ class DefaultOrchestrationProcessor implements OrchestrationProcessor {
                   try {
                     it.handle(event)
                   } catch (e) {
+                    log.warn("Error handling event (${event}): ${atomicOperation.class.simpleName}", e)
```

Comment:
threw this in here since it was annoying to trace the source of error without a call stack